docs/src/doc/dev/adr/0005-use-typescript.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation|dev/ADRs/5) Use typescript" />
+
+# 5) Use TypeScript
+
+Date: 2020-07-16
+
+## Status
+
+Accepted
+
+## Context
+The components library that implements the design system will be implemented in Vue (see ADR #1). In addition to that we disucssied what benefits we can gain by using [TypeScript](https://www.typescriptlang.org/).
+
+There are several products (as of date) in the WMDE realm that were build with Vue, TypeScript and [Decorators](https://github.com/vuejs/vue-class-component): Termbox, Tainted References, Data Bridge, and we anticipate there will be more. This means the components library needs to support TypeScript apps.
+
+Using TypeScript together with Decorators, when developing Vue projects, used to be the recomended way, because Decorators provide full compatibility between Vue and TypeScript. The TypeScript support in Vue has significantly improved since then.
```

Comment:
```suggestion
Using TypeScript together with Decorators, when developing Vue projects, used to be the recommended way, because Decorators provide full compatibility between Vue and TypeScript. The TypeScript support in Vue has significantly improved since then.
```